src/devices/HuskyLens/DataFrame.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
+
+namespace Iot.Device.HuskyLens
+{
+    internal class DataFrame
+    {
+        private readonly byte[] _data;
+
+        public DataFrame(ReadOnlySpan<byte> data)
+        {
+            _data = data.ToArray();
+        }
+
+        public Command Command
+        {
+            get
+            {
+                return (Command)_data[4];
+            }
+        }
+
+        public ReadOnlySpan<byte> Data
+        {
+            get
+            {
+                return new ReadOnlySpan<byte>(_data, 5, _data[3]);
+            }
+        }
+
+        public bool Valid()
```

Comment:
Better with `IsValid` as it's not a property but a method. In this case, starting with a verb is what is needed